android/anc/src/test/java/org/smartregister/fhircore/anc/AncConfigServiceTest.kt

```diff
@@ -58,7 +58,7 @@ class AncConfigServiceTest : RobolectricTest() {
 
   @Test
   fun testResourceSyncParam_shouldHaveResourceTypes() {
-    every { sharedPreferencesHelper.read(any(), any()) } returns
+    every { sharedPreferencesHelper.read(any(), null) } returns
```

Comment:
Is there a reason we are using null here?

Reply:
same reason as mentioned here
https://github.com/opensrp/fhircore/pull/1030#discussion_r805787445